src/qililab/analog/annealing_program.py

```diff
@@ -87,7 +87,9 @@ def _chip_element_to_short(self, chip_element: str) -> str:
             else f"{chip_element[0]}{split_element[1]}_{split_element[2]}"
         )
 
-    def get_waveforms(self, crosstalk_matrix: CrosstalkMatrix | None = None) -> dict[str, ArbitraryWave]:
+    def get_waveforms(
+        self, crosstalk_matrix: CrosstalkMatrix | None = None, minimum_clock_time: int = 1
+    ) -> dict[str, ArbitraryWave]:
         """Returns a dictionary containing (bus, waveform) for each flux control from the transpiled fluxes. `AnnealingProgram.transpile` should be run first. The waveform is an arbitrary waveform obtained from the transpiled fluxes.
 
         Args:
```

Comment:
Inside args please give a little bit of context about the minimum_clock_time (and why is it defaulted to 1)